<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageItem</class>
 <widget class="QWidget" name="MessageItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="profileText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="msgText">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>你好</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>friendName</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeText">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>4/25/24 20：00</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
